### ISSUE TITLE
new debug messages regarding regExp

### ISSUE THREAD

**Issue by octaeder** [texstudio-org/texstudio#3783]
### Environment
TeXstudio 4.8.3 (git 4.8.2-44-gad95c07be) private msys2 build
Using Qt Version 6.7.2, compiled with Qt 6.7.2 R
Win 10
Miktex

### Description
Currently regExp statements are being updated. I would like to note that since then I see many of the following messages

```
QRegularExpressionMatchIterator::next() called on an iterator already at end
```

for sufficiently large documents.

**Comment by octaeder** [comment]
maybe fixed with e52e30e0b (fix iterator warning)

**Comment by sunderme** [comment]
So, we can close this?
